### PR TITLE
Wire up batch delete

### PR DESCRIPTION
**Pull request by kdid**
Screen Shot 2021-02-10 at 6 50 54 AM
Screen Shot 2021-02-10 at 6 51 04 AM
Screen Shot 2021-02-10 at 6 51 28 AM
Screen Shot 2021-02-10 at 6 56 12 AM

### CODE REVIEW

**Comment by kdid** [comment]
@adamjarling - this is a draft. Do you mind a quick review?

File: assets/js/components/Search/BatchDeleteConfirmationModal.jsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    setBatchNickname(e.target.value);
+  };
+
+  const handleBatchDeleteConfirm = () => {
+    console.log("filteredQuery", filteredQuery);
+    console.log("nickname", batchNickname);
+    console.log("selectedItems", selectedItems);
+    let query = "";
+    if (selectedItems.length == 0) {
+      query = JSON.stringify({ query: filteredQuery });
```
Comment (adamjarling) [inline assets/js/components/Search/BatchDeleteConfirmationModal.jsx:64]: Could you remind me of how we'd get into this condition?

Comment (adamjarling) [reply assets/js/components/Search/BatchDeleteConfirmationModal.jsx:64]: Ah, nevermind... now I remember.

**Review by adamjarling — APPROVED** [review]
@kdid I think this looks great, and follows all the existing patterns to a T.  As the batch options on the Search screen grow, I think eventually we could look at a way to optimize and DRY up the code.  But this looks great, and I'd say just clean up the console logs and this is ready to go.

**Comment by kdid** [comment]
@adamjarling - undrafting - I also removed the view works button from the batch details page if it's a delete as well as hid the add/replace/delete areas.

Screen Shot 2021-02-10 at 12 53 44 PM

**Pull request merged**